Task3/src/main/java/Building.java

```diff
@@ -1,56 +1,71 @@
-import java.util.concurrent.Executors;
 import java.util.concurrent.locks.Condition;
 import java.util.concurrent.locks.Lock;
 import java.util.concurrent.locks.ReentrantLock;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+
 public final class Building {
 
+    @Getter
     private List<Floor> floors;
-
-    Controller controller;
-
+    @Getter
+    private Controller controller;
+    @Getter
     private Elevator elevator;
+    @Getter
+    private List<Passenger> passengers;
+    @Getter
+    private int passengersNumber;
+    @Getter
+    private Validators validators;
 
-    List<Passenger> passengersList;
 
     public Building() {
         initialize();
     }
 
-
     private void initialize() {
-        Properties prop = readPropertiesFile(
-                Objects.requireNonNull(Building.class.getResource("config.properties")).getPath());
-        int floorsNumber = Integer.parseInt(prop.getProperty("floorsNumber"));
-        int elevatorCapacity = Integer.parseInt(prop.getProperty("elevatorCapacity"));
-        int passengersNumber = Integer.parseInt(prop.getProperty("passengersNumber"));
-        initializePassengers(passengersNumber, floorsNumber);
+        PropertyUtil propertyUtil = new PropertyUtil("config.properties");
+        int floorsNumber = propertyUtil.getIntegerValueByPropertyName("floorsNumber");
+        int elevatorCapacity = propertyUtil.getIntegerValueByPropertyName("elevatorCapacity");
+        passengersNumber = propertyUtil.getIntegerValueByPropertyName("passengersNumber");
+        initializePassengers(floorsNumber);
         initializeFloors(floorsNumber);
         elevator = new Elevator(floorsNumber - 1, elevatorCapacity);
-        initializeController(passengersNumber, floorsNumber);
+        initializeController(floorsNumber);
+        initializeValidators();
     }
 
-    private void initializeController(int passengersNumber, int floorsNumber) {
+    private void initializeValidators() {
+        List<Validator> validatorList = List.of(
+                new ElevatorValidator(),
+                new DispatchContainersValidator(),
+                new TransportationStatesValidator(),
+                new PassengersValidator(),
+                new PassengersNumberValidator()
+        );
+        validators = new Validators(validatorList);
+    }
+
+    private void initializeController(int floorsNumber) {
         Lock lock = new ReentrantLock();
         Map<Integer, Condition> mapOfConditions =
                 Stream.iterate(0, n -> n + 1)
                         .limit(floorsNumber)
                         .collect(Collectors.toMap(i -> i, (i) -> lock.newCondition()));
-        controller = new Controller(floors, elevator, mapOfConditions, lock, passengersNumber);
+        controller = new Controller(floors, elevator, mapOfConditions, lock);
     }
 
-    private void initializePassengers(int passengersNumber, int floorsNumber) {
```

Comment:
Instead of assigning values to fields inside methods you can make methods which can return values and make something like this:
```
 @Getter
    private Controller controller = initializeController()
l```